hibernate-micrometer/src/test/java/org/hibernate/test/stat/MicrometerStatisticsTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.test.stat;
+
+import org.hibernate.Session;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.cfg.Environment;
+import org.hibernate.stat.HibernateMetrics;
+
+import org.hibernate.testing.junit4.BaseCoreFunctionalTestCase;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.search.MeterNotFoundException;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *  @author Erin Schnabel
+ *  @author Donnchadh O Donnabhain
+ */
+public class MicrometerStatisticsTest extends BaseCoreFunctionalTestCase {
+
+	@Override
+	protected Class<?>[] getAnnotatedClasses() {
+		return new Class<?>[] { Account.class, AccountId.class };
+	}
+
+	private SimpleMeterRegistry registry = new SimpleMeterRegistry();
+	private HibernateMetrics hibernateMetrics;
+
+	@Override
+	protected void configure(Configuration configuration) {
+		super.configure( configuration );
+
+		configuration.setProperty( Environment.USE_SECOND_LEVEL_CACHE, "false" );
```

Comment:
style nit on inconsistent use of space before/after first and last argument here and elsewhere in pr.